src/Microsoft.Health.Dicom.Api/Controllers/ChangeFeedController.cs

```diff
@@ -19,6 +19,8 @@
 
 namespace Microsoft.Health.Dicom.Api.Controllers
 {
+    [ApiController]
+    [ApiVersion("1.0-prerelease")]
```

Comment:
If we wanted, could we make the prerelease monitor another segment in the path, like `/api/v1.0/prerelease/`?

Reply:
I don't think this would work. In the ApiVersion class, the status (prerelease) must be appended as "-status". Doing "/status" will throw an error.
We could put it as part of the route, like "/api/v{version:apiVersion}/prerelease/" but then the version would be reported as just 1.0, and to add a new version we would need to register a new route instead of using the same one.